content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -0,0 +1,169 @@
+
+A `certified POS system <https://www.systemedecaisseenregistreuse.be/systemes-certifies>`_ must
+adhere to rigorous government regulations, which means it operates differently from a non-certified
+POS.
+
+- On a certified POS, you cannot:
+
+  - Make a refund.
+  - Set up and use the **global discounts** feature (the `pos_discount` module is blacklisted and
+    cannot be activated).
+  - Set up and use the **loyalty programs** feature (the `pos_loyalty` module is blacklisted and
+    cannot be activated).
+  - Reprint receipts (the `pos_reprint` module is blacklisted and cannot be activated).
+  - Modify prices in order lines.
+  - Modify or delete order lines in POS orders.
+  - Sell products without a valid VAT number.
+  - Use a POS that is not connected to an IoT box.
+
+- The :doc:`cash rounding <../../sales/point_of_sale/shop/cash_rounding>` feature must be
+  activated and set to a :guilabel:`Rounding Precision` of `0,05` and a :guilabel:`Rounding Method`
+  set as :guilabel:`Half-Up`.
+- Taxes must be set as included in the price. To set it up, go to :menuselection:`Point of Sale -->
+  Configuration --> Settings`, and from the :guilabel:`Taxes` section, open the :guilabel:`Default
+  Sales Tax` form by clicking the arrow next to the default sales tax field. There, click
+  :guilabel:`Advanced Options` and enable :guilabel:`Included in Price`.
+- At the start of a POS session, users must click :guilabel:`Work in` to clock in. Doing so allows
+  the registration of POS orders. If users are not clocked in, they cannot make POS orders.
+  Likewise, they must click :guilabel:`Work Out` to clock out at the end of the session.
+
+.. warning::
+   If you configure a POS to work with a black box, you cannot use it again without it.
+
+.. _belgium/fdm:
+
+The Fiscal Data Module
+----------------------
+
+The :abbr:`FDM (Fiscal Data Module)`, or `black box <https://www.xn--botenoire-r5a.be/fonctionnement>`_,
+is a government-certified device that works together with the Point of Sale application and saves
+your POS orders information. Concretely, a **hash** (:dfn:`unique code`) is generated for each POS
+order and added to its receipt. This allows the government to verify that all revenue is declared.
+
+.. note::
+   Ensure your black box is approved by the Belgian government. You can check the compliance of your
+   black box by visiting the `Service Public Federal Finances
+   <https://www.systemedecaisseenregistreuse.be/systemes-certifies#FDM%20certifi%C3%A9s>`__
+   website.
+
+Configuration
+~~~~~~~~~~~~~
+
+Before setting up your database to work with an FDM, ensure you have the following hardware:
+
+- a registered :ref:`black box <belgium/blackbox>` (go to `www.boîtenoire.be
+  <https://www.xn--botenoire-r5a.be/ma-caisse/>`_ to order yours);
+- a serial null modem cable per FDM (`example
+  <https://www.startech.com/Cables/Serial-Parallel-PS-2/DB9-DB25/10-ft-Cross-Wired-Serial-Null-Modem-Cable-DB9-FM~SCNM9FM>`__);
```

Comment:
Did you check this link?
You can use
https://www.startech.com/cables/scnm9fm
instead (I also removed the `en-us` bit from the link)